pyproject.toml

```diff
@@ -1759,6 +1759,7 @@ module = [
     "tests.sentry.mediators.token_exchange.test_refresher",
     "tests.sentry.mediators.token_exchange.test_validator",
     "tests.sentry.middleware.integrations.parsers.test_github",
+    "tests.sentry.middleware.integrations.parsers.test_gitlab",
```

Comment:
new files should not be blocklisted -- please fix the typing issues in this new file